woodchipper/http/flask.py

```diff
@@ -24,6 +24,12 @@ def wrapped_full_dispatch_request(self):
                 "body_size": request.content_length,
                 "method": request.method,
                 "url": request.url,
+                **{
+                    f"query_param.{param_key.lower()}": param_val_list[0]
```

Comment:
Let's go with `GET` instead of `query_param`?

Reply:
I don't follow? To clarify, for a request like `somedomain.com?qparam_key1=param_val`, you want the log context to hold `"http.GET.qparam_key1": "param_val"`?

Reply:
That would be my preference, yes.

Reply:
Why? We're already recording the method? It seems... weird to me to do it that way? Is there some queryability concern here?

This isn't hard to get around, but don't other methods allow query params? HEAD and OPTIONS I think. That would be workable by having `f"{request.method.upper()}...` though.

Reply:
In being challenged on it, I'm sure it has to do with my affinity for Django, not any real technical reason. Feel free to ignore this one.

Reply:
I left it as is, but I think there's value in deferring to the patterns (and naming) of urlparse. I like assuming the same standards/pattern of something in stdlib. Shove that entire six-tuple into the context.